website/client/components/chat/autoComplete.vue

```diff
@@ -80,7 +91,12 @@ export default {
       }
     },
     select (result) {
-      let newText = this.text.slice(0, this.currentSearchPosition + 1) + result;
+      let startIndex = this.carPos;
+      while (this.text[startIndex] !== '@' && startIndex >= 0) {
+        startIndex--;
+      }
+      startIndex++;
+      let newText = this.text.slice(0, this.currentSearchPosition) + result + this.text.slice(this.currentSearchPosition + index - startIndex);
```

Comment:
did you had another variable `index` during development? cause it doesn't exist anymore ^^

```
Uncaught ReferenceError: index is not defined
    at VueComponent.select (webpack-internal:///2130:103)
    at click (webpack-internal:///2348:18)
    at invoker (webpack-internal:///27:2022)
    at HTMLDivElement.fn._withTask.fn._withTask (webpack-internal:///27:1821)
```